xblock/reference/plugins.py

```diff
@@ -171,3 +130,42 @@ def __set__(self, xblock, value):
         We don't support this until we figure out what this means
```

Comment:
It was my understanding that the point of this pull request was to move the filesystem field into supported status. I'm confused how that can be done when the __set__ method for the field remains unsupported.